### PR TITLE
#734 : Http Commands Error Messages Test

### PR DESCRIPTION
**Pull request by svkaizoku**
Fixes: https://github.com/DiceDB/dice/issues/734

I will be splitting the implementation of this [issue](https://github.com/DiceDB/dice/issues/734) in multiple PR's. This PR contains tests similar to `check_type_test.go`
Note: There are changes to extra files due to linting

### CODE REVIEW

**Comment by svkaizoku** [comment]
@lucifercr07 @pratikpandey21 , feel free to review this whenever free. Thanks 😄

**Comment by apoorvyadav1111** [comment]
Hi @svkaizoku, thanks for the changes. I have reviewed the HTTP test file. However, I suggest reverting the changes due to code indentation. Those changes deviate from the primary objective of the PR and its issue. We can take that up as a separate issue or PR if needed. This is just a suggestion. 
Thanks.

**Comment by svkaizoku** [comment]
> Hi @svkaizoku, thanks for the changes. I have reviewed the HTTP test file. However, I suggest reverting the changes due to code indentation. Those changes deviate from the primary objective of the PR and its issue. We can take that up as a separate issue or PR if needed. This is just a suggestion. Thanks.

Hey @apoorvyadav1111 , thanks for the suggesstion. I was just following the contributing guidelines https://github.com/DiceDB/dice/blob/master/CONTRIBUTING.md#code-formatting. Maybe we should just format the files that we change. Maybe we should add that explicitly to the documentation. 

I have reverted the changes btw. Feel free to take a look at it again.

File: integration_tests/commands/http/check_type_test.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			exec.FireCommand(HTTPCommand{Command: "DEL", Body: map[string]interface{}{"key": "foo"}})
+			for i, cmd := range tc.commands {
+				if tc.delays[i] > 0 {
+					time.Sleep(tc.delays[i])
+				}
+				result, _ := exec.FireCommand(cmd)
+				if len(tc.assertType) == 0 {
```
Comment (apoorvyadav1111) [inline integration_tests/commands/http/check_type_test.go:114]: Thanks for taking up my suggestion. Yes, the code format for the file was not as per the requirements, but there are lot of other PRs with the same change, I am sure one of those will have it fixed before this is merged. I have question with this` if ` condition, given that none of the test cases have an empty array.
Thanks

**Review by apoorvyadav1111 — COMMENTED** [review]
Looks good to me. Left one clarifying question.

File: integration_tests/commands/http/check_type_test.go (hunk 1 above)
Comment (svkaizoku) [reply integration_tests/commands/http/check_type_test.go:114]: Thanks for the clarification. Let me remove that if here also. Thanks for pointing out.

Comment (svkaizoku) [reply integration_tests/commands/http/check_type_test.go]: I will still be keeping the switch case for the asserts for furture use cases.

**Review by JyotinderSingh — APPROVED** [review]

**Pull request merged**